example/nBody/src/Kernels.hpp

```diff
@@ -92,7 +92,7 @@ namespace alpaka::example::nBody
 
                             auto const distSqr = (distanceVector * distanceVector).sum() + EPS;
                             auto const distSixth = distSqr * distSqr * distSqr;
-                            auto const invDistCube = BaseType{1.0} / math::sqrt(distSixth);
+                            auto const invDistCube = BaseType{1.0} / math::rsqrt(distSixth);
```

Comment:
bug: it must be `auto const invDistCube = math::rsqrt(distSixth);` because `rsqrt() == 1 /sqrt()`

Reply:
oh, i see, from the comments on the function i thought the difference to sqrt was just that it ignores when the argument is negative